graphql/schema/types/performer.graphql

```diff
@@ -0,0 +1,196 @@
+enum GenderEnum {
+  MALE
+  FEMALE
+  TRANSGENDER_MALE
+  TRANSGENDER_FEMALE
+  INTERSEX
+}
+
+enum FakeTitsEnum {
+  YES
+  NO
+  NA
+}
+
+type Measurements {
+  cup_size: String
+  # TODO - should these be in cm to be consistent with height?
+  bust: Int
+  waist: Int
+  hip: Int
+}
+
+input MeasurementsInput {
+  cup_size: String
+  # TODO - should these be in cm to be consistent with height?
+  bust: Int
+  waist: Int
+  hip: Int
+}
+
+type Performer {
+  id: ID!
+  name: String!
+  disambiguation: String
+  aliases: [String!]!
+  gender: GenderEnum
+  urls: [URL!]!
+  birthdate: Date
```

Comment:
I think it would be smart to support fuzzy dates, so we can say that someone was born in 1998 if we don't know the month/date. There are multiple ways to support that, we can either use a string field, so "1998", or "1998-04" or "1998-04-25". Or we can use a Date field, and an additional enum `DateAccuracy` field that specifies whether the date is accurate to the year, month or date. For querying purposes it's easiest to just always return the string variant. 

I would also argue we should just add an age field that calculates on the fly based on birthdate.